projects/1.20-fabric/assets/caffeinated/caffeinated/lang/zh_cn.json

```
@@ -0,0 +1,42 @@
+{
+  "advancements.husbandry.brew_coffee.title": "在咖啡的味道中醒来",
+  "advancements.husbandry.brew_coffee.description": "在被架在营火上的炼药锅中煮瓶咖啡",
+
+  "block.caffeinated.coffee_bean_block": "咖啡豆块",
+  "block.caffeinated.coffee_berry_crate": "箱装咖啡果",
+  "block.caffeinated.ground_coffee_block": "磨制咖啡豆块",
+
+  "block.minecraft.banner.caffeinated.java.black": "黑色Java徽标",
+  "block.minecraft.banner.caffeinated.java.blue": "蓝色Java徽标",
+  "block.minecraft.banner.caffeinated.java.brown": "棕色Java徽标",
+  "block.minecraft.banner.caffeinated.java.cyan": "青色Java徽标",
+  "block.minecraft.banner.caffeinated.java.gray": "灰色Java徽标",
+  "block.minecraft.banner.caffeinated.java.green": "绿色Java徽标",
+  "block.minecraft.banner.caffeinated.java.light_blue": "淡蓝色Java徽标",
+  "block.minecraft.banner.caffeinated.java.light_gray": "淡灰色Java徽标",
+  "block.minecraft.banner.caffeinated.java.lime": "黄绿色Java徽标",
+  "block.minecraft.banner.caffeinated.java.magenta": "品红色Java徽标",
+  "block.minecraft.banner.caffeinated.java.orange": "橙色Java徽标",
+  "block.minecraft.banner.caffeinated.java.pink": "粉红色Java徽标",
+  "block.minecraft.banner.caffeinated.java.purple": "紫色Java徽标",
+  "block.minecraft.banner.caffeinated.java.red": "红色Java徽标",
+  "block.minecraft.banner.caffeinated.java.white": "白色Java徽标",
+  "block.minecraft.banner.caffeinated.java.yellow": "黄色Java徽标",
+
+  "effect.caffeinated.caffeine": "咖啡因",
+
+  "item.caffeinated.coffee_beans": "咖啡豆",
+  "item.caffeinated.coffee_berries": "咖啡果",
+  "item.caffeinated.coffee_bottle": "咖啡瓶",
+  "item.caffeinated.ground_coffee": "磨制咖啡豆",
+  "item.caffeinated.java_banner_pattern": "旗帜图案",
+  "item.caffeinated.java_banner_pattern.desc": "Java徽标",
+  "item.caffeinated.tiramisu": "提拉米苏",
+
+  "subtitles.block.cauldron.brew": "炼药锅：烹煮",
+  "subtitles.block.cauldron.bubble": "炼药锅：冒泡",
+  "subtitles.item.coffee_bottle.drink": "咕噜咕噜",

```

```suggestion
  "subtitles.item.coffee_bottle.drink": "饮用",
```
https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95
https://minecraft.fandom.com/wiki/Subtitles
见`所有实体` / `All Mobs`一栏惯用译法，可以任选其他词